### PR TITLE
[Jetchat] Update to snapshot 7141639

### PR DESCRIPTION
**Pull request by JoseAlcerreca**

### CODE REVIEW

File: Jetchat/app/src/main/java/com/example/compose/jetchat/NavActivity.kt [hunk 1]
```diff
@@ -56,66 +70,130 @@ class NavActivity : AppCompatActivity() {
...
+    private fun findNavController(): NavController {
+        val navHostFragment =
+            supportFragmentManager.findFragmentById(R.id.nav_host_fragment) as NavHostFragment
+        return navHostFragment.navController
+    }
+}
 
-                        // AndroidViewBinding(ContentMainBinding::inflate)
+@Composable
+fun <T : ViewBinding> FragmentAwareAndroidViewBinding(
```
Comment (chrisbanes) [inline Jetchat/app/src/main/java/com/example/compose/jetchat/NavActivity.kt:138]: Add comment linking to the bug again

Comment (chrisbanes) [reply Jetchat/app/src/main/java/com/example/compose/jetchat/NavActivity.kt:138]: Also, move this out to a seperate file.

File: Jetchat/app/src/main/java/com/example/compose/jetchat/NavActivity.kt [hunk 2]
```diff
@@ -56,66 +70,130 @@ class NavActivity : AppCompatActivity() {
...
                 }
             }
         }
     }
+}
 
-    override fun onSupportNavigateUp(): Boolean {
-        val navController = findNavController(R.id.nav_host_fragment)
-        return navController.navigateUp() || super.onSupportNavigateUp()
+private fun findFragmentContainerViews(
```
Comment (chrisbanes) [inline Jetchat/app/src/main/java/com/example/compose/jetchat/NavActivity.kt:188]: Ditto, move this to a seperate file

File: Jetchat/buildSrc/src/main/java/com/example/compose/jetchat/buildsrc/dependencies.kt [hunk 3]
```diff
@@ -52,12 +52,12 @@ object Libs {
...
 
         object Activity {
-            const val activityCompose = "androidx.activity:activity-compose:1.3.0-alpha02"
+            const val activityCompose = "androidx.activity:activity-compose:1.3.0-SNAPSHOT"
         }
 
         object Compose {
-            const val snapshot = ""
-            const val version = "1.0.0-alpha12"
+            const val snapshot = "7141639"
```
Comment (yrezgui) [inline Jetchat/buildSrc/src/main/java/com/example/compose/jetchat/buildsrc/dependencies.kt:59]: `7154166` is the new snapshot

File: Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/UserInput.kt [hunk 4]
```diff
@@ -385,25 +384,11 @@ private fun UserInputText(
     onTextFieldFocused: (Boolean) -> Unit,
     focusState: Boolean
 ) {
-    // Grab a reference to the keyboard controller whenever text input starts
```
Comment (yrezgui) [inline Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/UserInput.kt:388]: This logic isn't needed anymore?

**Review by yrezgui — APPROVED** [review]

File: Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/UserInput.kt (hunk 4 above)
Comment (JoseAlcerreca) [reply Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/UserInput.kt:388]: Nope

File: Jetchat/buildSrc/src/main/java/com/example/compose/jetchat/buildsrc/dependencies.kt (hunk 3 above)
Comment (JoseAlcerreca) [reply Jetchat/buildSrc/src/main/java/com/example/compose/jetchat/buildsrc/dependencies.kt]: yes

File: Jetchat/app/src/main/java/com/example/compose/jetchat/NavActivity.kt (hunk 2 above)
Comment (JoseAlcerreca) [reply Jetchat/app/src/main/java/com/example/compose/jetchat/NavActivity.kt]: done

File: Jetchat/app/src/main/java/com/example/compose/jetchat/NavActivity.kt (hunk 1 above)
Comment (JoseAlcerreca) [reply Jetchat/app/src/main/java/com/example/compose/jetchat/NavActivity.kt]: done

**Review by chrisbanes — APPROVED** [review]

**Pull request merged**